sound/soc/sof/loader.c

```diff
@@ -492,6 +501,12 @@ int snd_sof_load_firmware_raw(struct snd_sof_dev *sdev)
 			fw_filename);
 	}
 
+	/*
+	 * by default firmware should be loaded in a whole,
+	 * so don't skip the beginning during loading FW to DSP memory.
+	 */
```

Comment:
@ktrzcinx I think it is better to say probably say something like: "There is no extended manifest by default". But is this really needed at all. Isnt it initialized to 0 automatically?

Reply:
Yea, removed this part at all.